source/process/marketing-guidelines.rst

```diff
@@ -0,0 +1,68 @@
+============================================================
+Mattermost Marketing Content Guidelines
+============================================================
+
+How to write purposefully and persuasively 
+==============================================================
+
+In this document we set out some easy-to-follow guidelines about writing. Follow them, while also relying on your own judgment. Writing for blogs, newsletters, magazine articles and other forms of content will then become a joy, not a chore. 
```

Comment:
"Writing ... will then become a joy, not a chore" comes off as a little prescriptive / telling people how to feel